ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -189,7 +190,15 @@ public void onResponse(Response response) throws IOException {
               callback.invoke(0, null, e.getMessage());
               return;
             }
-            callback.invoke(response.code(), null, responseBody);
+
+            WritableMap responseHeaders = Arguments.createMap();
+            Headers headers = response.headers();
+
+            for (int i = 0; i < headers.size(); i++) {
+              responseHeaders.putString(headers.name(i), headers.value(i));
```

Comment:
We'll lose data if there are multiple values for the same header. okhttp has a method to get a list of values for a single header, for example.